client/js/app.modules/main/main-page-info.factory.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @ngInject
+ * */
+ module.exports = function (){
+     return {
+        title: 'Work',
+        isProjectView: true
```

Comment:
isProjectView used only for adding classes. I think you should delete it and do the same by adding class when you use this directive. (+header shouldn't be aware of such kind of data)